reframe/frontend/cli.py

```diff
@@ -733,6 +746,28 @@ def print_infoline(param, value):
             printer.debug(str(e))
             raise
 
+        printer.debug('Unusing module paths from command line')
+        for d in site_config.get('general/0/unuse_module_paths'):
+            try:
+                rt.modules_system.searchpath_remove(d)
+            except errors.EnvironError as e:
```

Comment:
I don't think that this is needed. Adding/removing paths does not throw any errors. If paths do not exist, it's harmless.

Reply:
If a module path is not defined yet removing a path returns an error.